README.md

```diff
@@ -45,7 +45,7 @@ and extract it to a new folder.
 Open a terminal window.
 
 Go to the folder when you extracted the source code. On a Mac, you _probably_ can do this by entering this command into
-the terminal: `cd Downloads/helloworld-telegram-chat-bot-master`
```

Comment:
we will use the download functionality from GitHub
when one downloads instead of cloning, that is the name of the folder